submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,367 @@
+
+## Solid fuel vs. rocket fuel
+
+Having seen that converting coal into solid fuel offers a significant energy yield, perhaps the same is true about making rocket fuel? To explore this, instead of starting from the beginning with coal/oil, it is enough to just look at the recipe for rocket fuel:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing solid fuel into rocket fuel'
+ caption='Processing solid fuel into rocket fuel'
+
+%}
+
+We see that a single unit rocket fuel contains 100MJ of energy while the required ingredients of 10 solid fuel already holds 120MJ, and the 10 light oil can be made into an 11th solid fuel to provide almost 12MJ more. Hence, we see that processing solid fuel into rocket fuel makes you lose energy, before even accounting for processing energy. However, as we will cover later on, rocket fuel has advantages over solid fuel that may outweigh the energy losses. Furthermore, it is a necessary ingredient if we want to make nuclear fuel, which may increase the overall energy gains despite the losses at this stage.
+
+## Going nuclear
+
+Nuclear fuel is still pretty experimental in the real world, but Factorio offers a standardized variant that is made from regular rocket fuel and uranium-235 and it presents itself as the ultimate burner fuel. Like with our examination of rocket fuel, the recipe itself is enough to determine energy gains:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing rocket fuel into nuclear fuel'
+ caption='Processing rocket fuel into nuclear fuel'
+
+%}
+
+We see that nuclear fuel offers an impressive energy output of 1.21GJ, which is [a reference](https://wiki.factorio.com/Nuclear_fuel), but also around 10 times as much energy as the solid fuel first used to make the rocket fuel. This means that if you were to switch from burning coal directly to making it into nuclear fuel to burn instead, your coal demand would drop to a tiny fraction, to as little as 1/20 as before. 
+
+## Making the most of your uranium
+
+Despite its massive energy gains and the resulting coal and oil savings, nuclear fuel might not be perfect. The uranium-235 that it uses could instead be used to make uranium fuel cells, which provide a different energy situation: They can only be used inside nuclear reactors. One fuel cell provides one reactor a constant supply of 40MW of heat for exactly 200 seconds, which adds up to a total energy of `40MW * 200s = 8000MJ = 8GJ`. If the reactor has any neighbors, the neighbor bonus kicks in, and the bonus from each neighbor leads to an additional 40MW of heat being supplied by the same fuel cell. Since the reactor can up to 4 neighbors, the fuel cell can provide up to 5 times its original yield, which is `8GJ * 5 = 40GJ`. We already dealing with GJ instead of MJ when discussing a single fuel cell but the recipe for them yields 10 per piece of uranium-235:
```

Comment:
```suggestion
Despite its massive energy gains and the resulting coal and oil savings, nuclear fuel might not be perfect. The uranium-235 that it uses could instead be used to make uranium fuel cells, which provide a different energy situation: They can only be used inside nuclear reactors. One fuel cell provides one reactor a constant supply of 40MW of heat for exactly 200 seconds, which adds up to a total energy of `40MW * 200s = 8000MJ = 8GJ`. If the reactor has any neighbors, the neighbor bonus kicks in, and the bonus from each neighbor leads to an additional 40MW of heat being supplied by the same fuel cell. Since the reactor can have up to 4 neighbors, the fuel cell can provide up to 5 times its original yield, which is `8GJ * 5 = 40GJ`. We already dealing with GJ instead of MJ when discussing a single fuel cell but the recipe for them yields 10 per piece of uranium-235:
```